client/src/components/ElementEditor/Header.js

```diff
@@ -1,14 +1,93 @@
 import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
 
 class Header extends Component {
   render() {
+    const { title, elementType, fontIcon, moreActions, modified, draft } = this.props;
+
+    const spanClassNames = classNames({
+      'element-item--modified': modified,
+      'element-item--draft': draft
+    });
+
+    const editClassNames = classNames(
+      'element-editor__icon-action',
+      'element-editor__icon-action--hidden-on-hover',
+      'font-icon-edit btn--icon-large action-menu--handled',
+      'action action-detail',
+      'edit-link',
+    );
+
+    const deleteClassNames = classNames(
+    'action',
+    'gridfield-button-delete',
+    'btn--icon-md',
+    'font-icon-trash-bin',
+    'btn--no-text',
+    'element-editor__icon-action',
+    'action-menu--handled',
+    'form-group--no-label',
+    'gridfield-button-delete font-icon-trash',
+     );
+
     return (
-      <div />
+      <div className="element__heading">
+        <div className="element-header__info">
+          <div className="element-preview__icon">
+            <i className={fontIcon} />
+            {
+              modified &&
+              <span className={spanClassNames} title="Item has unpublished changes" />
+            }
+          </div>
+          <h3 className="element-heading__title">{title}
+            <small className="element-heading__element-type">{ elementType }</small>
+          </h3>
+        </div>
+        <div className="element-header__actions">
+          <a
+            className={editClassNames}
+            title="Edit"
+          >
+            <span className="sr-only">
+              Edit
+            </span>
+          </a>
+          <button
+            type="submit"
+            className={deleteClassNames}
+            aria-label="Delete"
+            title="Delete"
+            data-url="admin/pages/edit/EditForm/8/field/ElementalArea"
```

Comment:
We'll need to inject this URL somehow, it can't really contain hard coded URLs or IDs